dids/src/main/kotlin/web5/sdk/dids/DidMethod.kt

```diff
@@ -171,36 +168,16 @@ public interface DidMethod<T : Did, O : CreateDidOptions> {
   public fun load(uri: String, keyManager: KeyManager): T
 }
 
-/**
- * Finds the first available assertion method from the [DIDDocument]. When [assertionMethodId]
- * is null, the function will return the first available assertion method.
- */
-@JvmOverloads
-public fun DIDDocument.findAssertionMethodById(assertionMethodId: String? = null): VerificationMethod {
```

Comment:
moved to [here](https://github.com/TBD54566975/web5-kt/pull/248/files#diff-7f7f709308a3c318c3c2ac2f3a4d7d68474ca11929c60e59757c9f656812b6c9R151)